Modules/Dashboard/Http/Controllers/DashboardWebController.php

```
@@ -0,0 +1,114 @@
+     * View a specific report
+     */
+    public function viewReport(Request $request, int $id)
+    {
+        // Logic to view report would go here
+        return view('dashboard::view-report', ['reportId' => $id]);
+    }
+
+    /**
+     * Download a report file
+     */
+    public function downloadReport(Request $request, int $id, string $filename)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (Unused Code Rules)

(UnusedFormalParameter)